src/main/groovy/com/etendoerp/legacy/dependencies/container/ArtifactDependency.groovy

```diff
@@ -24,6 +24,19 @@ class ArtifactDependency {
             PublicationUtils.ETENDO,
             PublicationUtils.BASE_MODULE_DIR
     )
+    
+    public static String JAR_ETENDO_MODULE_LOCATION_WOUT_ESCAPED_CHARS = PathUtils.createPathWithCustomSeparator(
+            "/",
```

Comment:
Check SO to use the correct path separator.

Reply:
I created PathUtils.createPathWith Custom Separator just to customize the separator, this variable is used to avoid PatternSyntaxException, it should not be constructed around the system separator